docs/reference/cooperative_groups_reference.rst

```diff
@@ -0,0 +1,92 @@
+.. meta::
+  :description: This chapter lists  types and device API wrappers related to the Cooperative Group
+                feature. Programmers can directly use them in their kernels to make use of this feature.
+  :keywords: AMD, ROCm, HIP, cooperative groups
+
+*******************************************************************************
+Cooperative Groups API
+*******************************************************************************
+
```

Comment:
Perhaps we should include something like this so we have a cross reference between reference and How To.
this should be a standard. 

This topic lists and describes the cooperative groups functions. For more information about how to use cooperative groups, see https://advanced-micro-devices-demo--3478.com.readthedocs.build/projects/HIP/en/3478/how-to/cooperative_groups.html.

Perhaps a link in the How to to the Reference, too.

Reply:
Added references.